.github/workflows/run_tests.yml

```diff
@@ -1,32 +1,45 @@
 on:
   pull_request:
-    branches:
-    - master
-    - main
-    - develop
     types:
       - opened
       - reopened
       - synchronize
       - closed
 
+
+
 jobs:
-  install_kbsdk_and_test:
+  install_static_narrative_and_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.8', '3.9', '3.10', '3.11']
```

Comment:
Do we need to test on all versions?
Okay, maybe, since I'm not even sure what Python version the base image is anymore. Hopefully >= 3.8.